crates/fluvio-spu-schema/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-spu-schema"
-version = "0.15.0"
+version = "0.15.1"
 edition = "2021"
```

Comment:
this probably needs to be bump up to `0.16.0`

Reply:
Sure!